### PR TITLE
♻️Maintenance/remove minio-sdk

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- [x] totally removes the use of python minio library
- [x] removes filestash/onlyoffice from the docker-compose-ops stack as they are useless (minio has its own UI)

## Related issue/s

- requires https://github.com/ITISFoundation/osparc-simcore/pull/5108

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by GitHK — COMMENTED** [review]
Cool. I did not see if you removed onlyoffice and fileshtash in yaml files. Did I miss it?

File: packages/pytest-simcore/src/pytest_simcore/helpers/utils_envs.py [hunk 1]
```diff
@@ -29,12 +29,17 @@ def setenvs_from_dict(
...
+            value is not None
+        ), f"{key=}{value=}"  # None keys cannot be is defined w/o value
+        converted_value = value
+        if isinstance(value, bool):
+            converted_value = f"{'true' if value else 'false'}"
         assert isinstance(
-            value, str
-        ), "client MUST explicitly stringify values since some cannot be done automatically e.g. json-like values"
+            converted_value, str
+        ), f"client MUST explicitly stringify values since some cannot be done automatically e.g. json-like values. problematic {key=}{value=}"
```
Comment (bisgaard-itis) [inline packages/pytest-simcore/src/pytest_simcore/helpers/utils_envs.py:40]: same here

**Review by bisgaard-itis — APPROVED** [review]
Cool! Thanks a lot for cleaning up. I have a few comments/questions.

File: packages/pytest-simcore/src/pytest_simcore/aws_server.py [hunk 2]
```diff
@@ -71,16 +73,22 @@ def mocked_ec2_server_envs(
...
 @pytest.fixture
-async def mocked_s3_server_envs(
-    mocked_aws_server: ThreadedMotoServer,
-    reset_aws_server_state: None,
+def mocked_s3_server_settings(
+    mocked_aws_server: ThreadedMotoServer, reset_aws_server_state: None, faker: Faker
+) -> S3Settings:
+    return S3Settings(
+        S3_ACCESS_KEY="xxx",
+        S3_ENDPOINT=f"http://{mocked_aws_server._ip_address}:{mocked_aws_server._port}",  # pylint: disable=protected-access # noqa: SLF001
```
[the remark this answers is not in the document]
Comment (sanderegg) [reply packages/pytest-simcore/src/pytest_simcore/aws_server.py:81]: these are private in the moto library. at the time I did not find any other way to get them

File: packages/pytest-simcore/src/pytest_simcore/helpers/utils_envs.py [hunk 3]
```diff
@@ -29,12 +29,17 @@ def setenvs_from_dict(
 ) -> EnvVarsDict:
     for key, value in envs.items():
         assert isinstance(key, str)
-        assert value is not None  # None keys cannot be is defined w/o value
+        assert (
+            value is not None
+        ), f"{key=}{value=}"  # None keys cannot be is defined w/o value
```
[the remark this answers is not in the document]
Comment (sanderegg) [reply packages/pytest-simcore/src/pytest_simcore/helpers/utils_envs.py:34]: you are absolutely right. but actually the = makes them come apart. but I will correct. thanks!

**Comment by sanderegg** [comment]
> Cool. I did not see if you removed onlyoffice and fileshtash in yaml files. Did I miss it?

yep good catch. commit missing. now done

**Review by pcrespov — APPROVED** [review]
oh boy! 🎉  thx a lot!

**Review by GitHK — COMMENTED** [review]
very good 👍

**Pull request merged**